docs/source/llm_recipes.md

```diff
@@ -0,0 +1,26 @@
+LLM Quantization Models and Recipes
+---
+
+Intel® Neural Compressor supported advanced large language models (LLMs) quantization technologies including SmoothQuant (SQ) and Weight-Only Quant (WOQ), 
+and verified a list of LLMs on 4th Gen Intel® Xeon® Scalable Processor (codenamed Sapphire Rapids) with [PyTorch](https://pytorch.org/) and [Intel® Extension for PyTorch](https://github.com/intel/intel-extension-for-pytorch). 
```

Comment:
In fact, WOQ is verified with ITREX and SQ is verified with Torch/IPEX, we may need to mention seperately or add ITREX here if we'd like to mentioned it ambiguously